relayer/src/chain.rs

```diff
@@ -165,7 +165,7 @@ pub trait ChainEndpoint: Sized {
         &self,
         client_id: &ClientId,
         height: ICSHeight,
-    ) -> Result<Self::ClientState, Error>;
+    ) -> Result<AnyClientState, Error>;
```

Comment:
I think the same change should be made for ``query_upgraded_client_state``

Reply:
https://github.com/informalsystems/ibc-rs/pull/1625/commits/9278d0460b926be2c640307302bb1d4ce8932f25